### ISSUE TITLE
[Accessibility] Show Code Actions menu item selected state not being announced correctly in screenreader

### ISSUE THREAD

**Issue by gdomingu** [microsoft/vscode#231070]
## Environment Details:
Application Name: VS Code
Visual studio code: 1.94.2 (Universal)
Copilot chat: v0.21.1
Windows 11 Home 23H2

## Actual Result:​
Screenreader (NVDA, VoiceOver) announces `not selected`. When navigating on code actions menu.

## Expected Result:​
Screenreader should not announce `not selected` nor `selected` for single select box.

## Repro steps:

1. Select a code snippet.
2. Open the code actions menu by clicking the lightbulb button or using (control + . / command + .)
3. Navigate up and down the menu using the up and down keys.
4. Hear the screenreader announce `not selected`.

![Image](https://github.com/user-attachments/assets/9617ef93-09c6-4b5f-8a3d-4868b312100d)

**Comment by meganrogge** [comment]
I notice you're not in screen reader optimized mode. If you enable that (with the toggle screen reader optimized mode command), does this change?

**Comment by meganrogge** [comment]
related: https://github.com/microsoft/vscode/issues/152228

**Comment by meganrogge** [comment]
see https://github.com/microsoft/vscode/issues/152228#issuecomment-2411661926 with the recommendation to close the issue as invalid.

**Comment by gdomingu** [comment]
> I notice you're not in screen reader optimized mode. If you enable that (with the toggle screen reader optimized mode command), does this change?

@meganrogge No it does not change even when in screen reader optimized mode.

![Image](https://github.com/user-attachments/assets/b2cdc29a-78dc-44db-9062-9f24bf1f781c)

**Comment by meganrogge** [comment]
Listening to screen reader users on this one, I think this shld be closed. Their argument abt the existing behavior in run and debug also applies here.
